engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlan.java

```diff
@@ -1359,7 +1409,8 @@ private void addDAGToCurrentDAG(ModuleMeta moduleMeta)
     String name;
     for (OperatorMeta operatorMeta : subDag.getAllOperators()) {
       name = subDAGName + MODULE_NAMESPACE_SEPARATOR + operatorMeta.getName();
-      this.addOperator(name, operatorMeta.getOperator());
+      Operator op = this.addOperator(name, operatorMeta.getOperator());
```

Comment:
OperatorMeta has name and id declared as final field. In our case we want to add the operator to top level dag with expanded name and wants to have a unique id in the parent DAG. Hence reused the addOperator api without adding a new one. This has additional cost of allocating new OperatorMeta though.